src/storm/settings/modules/MinMaxEquationSolverSettings.cpp

```diff
@@ -146,6 +168,18 @@ bool MinMaxEquationSolverSettings::isForceIntervalIterationSymmetricUpdatesSet()
     return this->getOption(intervalIterationSymmetricUpdatesOptionName).getHasOptionBeenSet();
 }
 
+bool MinMaxEquationSolverSettings::getLpUseOnlyInitialStateAsObjective() const {
+    return this->getOption(lpOptimizeOnlyInitialStateOptionName).getArgumentByName("optimization-type").getValueAsString() == "onlyinitial";
+}
+
+bool MinMaxEquationSolverSettings::getLpUseNonTrivialBounds() const {
+    return this->getOption(lpEqualityForUniqueActionsOptionName).getHasOptionBeenSet();
```

Comment:
```suggestion
    return this->getOption(lpUseNonTrivialBoundsOptionName).getHasOptionBeenSet();
```

Reply:
Oops, yes this is the one flag that has not been actually used...